decred/decred/dcr/account.py

```diff
@@ -1952,14 +1956,13 @@ def sync(self):
         signals.balance(self.calcBalance())
         self.generateGapAddresses()
 
-        # If there is a chosen stake pool, sync the purchaseInfo.
-        # TODO: Save purchase info
+        # If there is a chosen stake pool, authorize the pool.
         stakePool = self.stakePool()
         if stakePool:
             try:
-                stakePool.getPurchaseInfo()
+                stakePool.authorize(self.votingAddress().string())
```

Comment:
@JoeGruffins I didn't see any problem calling `authorize` here instead of below. `authorize` calls `purchaseInfo` internally. Is this OK?

Reply:
Looks good.